From 620f885e15281ca32108331e68aec4ace5bdcec2 Mon Sep 17 00:00:00 2001
From: Juan Quintela <quintela@redhat.com>
Date: Tue, 14 Jan 2014 15:07:24 +0100
Subject: [PATCH 13/40] memory: use bit 2 for migration

RH-Author: Juan Quintela <quintela@redhat.com>
Message-id: <1389712071-23303-14-git-send-email-quintela@redhat.com>
Patchwork-id: 56668
O-Subject: [RHEL7 qemu-kvm PATCH 13/40] memory: use bit 2 for migration
Bugzilla: 997559
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

For historical reasons it was bit 3.  Once there, create a constant to
know the number of clients.

Signed-off-by: Juan Quintela <quintela@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Reviewed-by: Orit Wasserman <owasserm@redhat.com>
(cherry picked from commit 5adca7ace9b80bf594e4c52c0d5b23573cba3639)
Signed-off-by: Juan Quintela <quintela@trasno.org>
---
 include/exec/memory.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/exec/memory.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/exec/memory.h b/include/exec/memory.h
index 868fdb1..0023edf 100644
--- a/include/exec/memory.h
+++ b/include/exec/memory.h
@@ -32,7 +32,8 @@ typedef struct MemoryRegionMmio MemoryRegionMmio;
 
 #define DIRTY_MEMORY_VGA       0
 #define DIRTY_MEMORY_CODE      1
-#define DIRTY_MEMORY_MIGRATION 3
+#define DIRTY_MEMORY_MIGRATION 2
+#define DIRTY_MEMORY_NUM       3        /* num of dirty bits */
 
 struct MemoryRegionMmio {
     CPUReadMemoryFunc *read[3];
-- 
1.7.1

